cloud-compose/templates/docker-compose.override.yml

```diff
@@ -24,6 +24,9 @@ services:
       MONGODB_ADMIN_PASSWORD: {{ MONGODB_ADMIN_PASSWORD }}
     {%- endif %}
   mongodb:
+    {%- if mongo is defined and mongo.journal == False %}
```

Comment:
you need to check that mongo.journal is defined as well before checking the value